consensus/src/common.rs

```diff
@@ -167,17 +167,17 @@ mod tests {
         assert_eq!(epoch_slot, Some(2));
     }
 
-    #[test]
-    fn test_protocol_state_chain_state_hash() {
-        let mut c: ProtocolStateChain = ProtocolStateChain(vec![]);
+    // #[test]
```

Comment:
Are these commented accidentally?

Reply:
Oh yeah they are, There should be no changes to this file